Header header

string robot_name
string current_state
float64 current_utility
string assigned_table
#
float64 u_idle
float64 u_docker
float64 u_waiter
float64 u_home
#
float64[5] u_table
#float64 u_table0
#float64 u_table1
#float64 u_table2
#float64 u_table3
#float64 u_table4
#float64 u_table5
#float64 u_table6
 